### PR TITLE
cedar chart

### LINKED ISSUE #208: new demo: cedar chart
with map interaction reflected in the chart.

prior art: http://cedar-v0.surge.sh/examples/bar-map-leaflet-integration.html

### PR DESCRIPTION
**Pull request by gavinr**
initial cedar chart demo - tried to be consistent with [existing chart demo](https://esri.github.io/esri-leaflet/examples/dynamic-chart.html) for comparison sake.  chart responds to map pan/zoom, but hover on chart points does not change map view - i think we need https://github.com/Esri/cedar/issues/282 to be completed to allow for that. resolves #208 

would love your feedback please @jwasilgeo and @jgravois

### CODE REVIEW

File: src/pages/examples/cedar-chart.hbs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+</div>
+
+<script>
+  // STEP 1: MAKE A MAP AND ADD LAYERS
+  var map = L.map('map').setView([45.526, -122.667], 13);
+
+  L.esri.basemapLayer('Topographic').addTo(map);
+
+  // create and add a feature layer
+  // features from this layer will appear in the Chart.js scatterplot
```
[the remark this answers is not in the document]
Comment (gavinr) [reply src/pages/examples/cedar-chart.hbs]: copy/paste fail. thanks for the catch! 👍

File: src/pages/examples/cedar-chart.hbs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    chart = new cedar.Chart(elementId, {"type": "scatter"})
+      .datasets(datasets)
+      .series(series)
+      .overrides(overrides);
+
+    // render the chart
+    chart.show();
+
+    // only run this "createChart" function once - so remove the event listener:
+    treesFeatureLayer.off('load');
```
Comment (jwasilgeo) [inline src/pages/examples/cedar-chart.hbs]:
```suggestion
```
(only if deciding to replace with `once` above)

File: src/pages/examples/cedar-chart.hbs [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  // features from this layer will appear in the Chart.js scatterplot
+  var treesFeatureLayer = L.esri.featureLayer({
+    url: 'https://services.arcgis.com/rOo16HdIMeOBI4Mb/arcgis/rest/services/Heritage_Trees_Portland/FeatureServer/0',
+  });
+
+  treesFeatureLayer.addTo(map);
+
+  // STEP 2: DEFINE A CHART
+  var chart;
+  treesFeatureLayer.on('load', function() {
```
Comment (jwasilgeo) [inline src/pages/examples/cedar-chart.hbs]: Could `once` be used here to replace the `on` and subsequent `off` below?  Just an idea...not a big deal to me either way.

https://leafletjs.com/reference-1.4.0.html#evented-once

```suggestion
  treesFeatureLayer.once('load', function() {
```

File: src/partials/sidebar-examples.hbs [hunk 4]
```diff
@@ -103,6 +103,7 @@
     <a href="{{assets}}examples/getting-service-metadata.html">Getting Service Metadata</a>
     <a href="{{assets}}examples/parse-feature-collection.html">Parsing Feature Collections</a>
     <a href="{{assets}}examples/indoors.html">Indoor Mapping</a>
-    <a href="{{assets}}examples/dynamic-chart.html">Dynamic Chart</a>
+    <a href="{{assets}}examples/dynamic-chart.html">Dynamic Chart (Chart.js)</a>
+    <a href="{{assets}}examples/cedar-chart.html">Dynamic Chart (Cedar)</a>
```
[the remark this answers is not in the document]
Comment (jwasilgeo) [reply src/partials/sidebar-examples.hbs]: I agree with that.

**Pull request merged**